main/webapp/modules/core/scripts/views/data-table/menu-reconcile.js

```diff
@@ -598,6 +592,13 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
       click: doUseValuesAsIdentifiers
     },
     {},
+    {
+      id: "core/Add-column-with-URLs-of-matched-entities",
+      label: $.i18n("core-views/add-column-with-URLs-of-matched-entities"),
+      tooltip: $.i18n("core-views/add-column-with-URLs-of-matched-entities-tooltip"),
+      click:  doAddColumnWithUrlOfMatchedEntities
+    },
+    {},
```

Comment:
I think this separator is not needed, given that the two actions (for identifiers and URLs) are very similar: they can be in the same section of the menu.